x509util/x509util.go

```diff
@@ -862,7 +862,7 @@ func ExtractSCT(sctData *x509.SerializedSCT) (*ct.SignedCertificateTimestamp, er
 }
 
 // MarshalSCTsIntoSCTList serializes SCTs into SCT list.
-func MarshalSCTsIntoSCTList(scts []*ct.SignedCertificateTimestamp) (*x509.SignedCertificateTimestampList, error) {
+func MarshalSCTsIntoSCTList(scts []*ct.SignedCertificateTimestamp) ([]byte, error) {
```

Comment:
Why shouldn't this function return an x509.SignedCertificateTimestampList? That seems like a sensible thing for it to return. If the caller wants to marshal that into TLS, they can trivially do so. If you continue and change this function signature, that's a backwards-incompatible change that'll require a major version bump to certificate-transparency-go.

Reply:
Yes I didn't really have context for this change was was reviewing in isolation. Not sure it should be made now.

Reply:
outdated, moved back the initial interface.